### DISCUSSION TITLE
Left and Right hand brackets

### DISCUSSION THREAD

**Discussion by asutrem** [musescore/MuseScore#15495] [Ideas]
Is it possible to add the brackets that denote which hand to use for the piano? Like the ones seen in Henle scores.

**Comment by ghost** [comment]
Sort of: for now, open the master palette (Shift+F9) and under 'Symbols -> Keyboard techniques' you'll see symbols for these brackets:
![image](https://user-images.githubusercontent.com/20604032/209071763-0a7c1a59-a974-4be9-a0e3-2952baa0c6bb.png)
which you can apply to the score.
![image](https://user-images.githubusercontent.com/20604032/209071932-26483470-d9d8-42d5-852e-0626985f25dc.png)

It's not ideal since they have to be positioned manually, and have fixed dimensions. I'd like to implement a 'smart' version of these as soon as we can get to it.
